apollo-runtime/src/main/java/com/apollographql/apollo/cache/normalized/CacheControl.java

```diff
@@ -1,8 +1,33 @@
 package com.apollographql.apollo.cache.normalized;
 
+/**
+ * CacheControl class represents the cache controlling strategies for reading GraphQL responses back from the
+ * normalized cache when a network request is made.
```

Comment:
We should omit this `when a network request is made.` as for instance cache only we don't make network call at all.